attributes/default.rb

```diff
@@ -1,2 +1,3 @@
 default['scrutinizer-whitelist']['source-url'] = 'https://scrutinizer-ci.com/api/meta'
 default['scrutinizer-whitelist']['priority'] = '05'
+default['scrutinizer-whitelist']['ports'] = [22]
```

Comment:
note you'll obv have to have the cookbook depend on sshd cookbook for it to work when sshd cookbook not in run list